dvc/command/metrics.py

```diff
@@ -100,6 +105,59 @@ def run(self):
         return 0
 
 
+def _show_diff(diff):
+    from texttable import Texttable
+
+    if not diff:
+        return "No changes."
+
+    table = Texttable()
+
+    # remove borders to make it easier for users to copy stuff
+    table.set_chars(("", "", "", ""))
+    table.set_deco(0)
+
+    rows = [["Path", "Metric", "Value", "Change"]]
```

Comment:
@dmpetrov Asked for this format specifically, so don't be surprised that we don't show old value here :)